numpy/ma/tests/test_regression.py

```diff
@@ -47,7 +47,7 @@ def test_atleast_2d(self):
         # Ticket #1559
         a = np.ma.masked_array([0.0, 1.2, 3.5], mask=[False, True, False])
         b = np.atleast_2d(a)
-        assert_(a.mask.ndim == 1)
```

Comment:
This change indicates that something is not 100% backwards compatible in the updates I made to `ma`. I believe that the version I have here is the expected and correct one.

Reply:
The fact this used to be `1` scares me

Reply:
I found a number of interesting things in the ma wrapper classes when I first started doing this.

Reply:
I just realized I made a mistake here: `a.mask.ndim` absolutely should be 1`. And `b.mask.ndim` should be 2. Fixed on my end now. Whoops!